src/client/Microsoft.Identity.Client/ITokenCacheInternal.cs

```diff
@@ -66,16 +62,18 @@ internal interface ITokenCacheInternal : ITokenCache, ITokenCacheSerializer
         /// Shows if MSAL's in-memory token cache has any kind of RT or non-expired AT. Does not trigger a cache notification.
         /// Ignores ADAL's cache.
         /// </summary>
-        bool HasTokensNoLocks();
+        bool HasTokensNoLocks(string partitionKey);
 
 
-        bool IsTokenCacheSerialized();
+        /// <summary>
+        /// True when MSAL has been configured to fire the serialization events. This can be done by the app developer or by MSAL itself (on UWP)
+        /// </summary>
+        bool IsExternalSerializationEnabled();
 
         /// <summary>
-        /// MSAL adds serialziation for UWP and also for ConfidentialClient app token cache. 
-        /// If the app developer provides their own serialization, this flags is false
+        /// True when MSAL has been configured by the app developer to fire serialization events.
         /// </summary>
-        bool UsesDefaultSerialization { get; }
+        bool IsExternalSerializationConfigured();
```

Comment:
@bgavrilMS Maybe rename this to clarify the difference between the two methods: enabled vs configured? Enabled includes default serialization, while this one, configured, is only for user-enabled serialization? So something like `IsExternalSerializationUserEnabled`..